io.openems.edge.controller.battery.batteryprotection/readme.adoc

```diff
@@ -0,0 +1,3 @@
+= io.openems.edge.controller.battery.batteryprotection
```

Comment:
Fix Readme title + also add some description about the Controller. This information is used to generate https://openems.github.io/openems.io/openems/latest/edge/controller.html